src/SFA.DAS.ApprenticePortal.Web/Pages/Home.cshtml.cs

```diff
@@ -28,18 +30,19 @@ public async Task OnGet()
             try
             {
                 if (!_notifications.Notifications.Contains(HomeNotification.ApprenticeshipDidNotMatch))
-                    CourseName = await GetCourseName();
+                    await PopulateProperties();
             }
             catch
             {
                 _notifications.Notifications.Add(HomeNotification.ApprenticeshipDidNotMatch);
             }
         }
 
-        private async Task<string> GetCourseName()
+        private async Task PopulateProperties()

```

Comment:
This could be writtern as private aysnc Task<(string couseName, bool confirmed)> GetValues()